### PR TITLE
feature : include chat message id in result

### PR DESCRIPTION
**Pull request by tsiougkosn**
Return the chat message id in the api response.

### CODE REVIEW

File: packages/server/src/index.ts [hunk 1]
```diff
@@ -1814,7 +1814,8 @@ export class App {
             if (result?.sourceDocuments) apiMessage.sourceDocuments = JSON.stringify(result.sourceDocuments)
             if (result?.usedTools) apiMessage.usedTools = JSON.stringify(result.usedTools)
             if (result?.fileAnnotations) apiMessage.fileAnnotations = JSON.stringify(result.fileAnnotations)
-            await this.addChatMessage(apiMessage)
+            const chatMessage = await this.addChatMessage(apiMessage)
+            result.chatMessageId = chatMessage.id
```
Comment (HenryHengZJ) [inline packages/server/src/index.ts:1818]: do you need the `chatId` (similar to sessionid) or the id of the message?
FYI - the latest changes return chatId along with other data - https://github.com/FlowiseAI/Flowise/pull/1586

Comment (tsiougkosn) [reply packages/server/src/index.ts:1818]: We need the message id, so we can keep feedback on the specific response. Else we need to have a second api call to /chatmessage in order DESC to get the last message, but this is not efficient

Comment (tsiougkosn) [reply packages/server/src/index.ts:1818]: Our end goal is to have map the message with the trace in Langfuse

Comment (HenryHengZJ) [reply packages/server/src/index.ts:1818]: got it!

**Review by HenryHengZJ — APPROVED** [review]

**Pull request merged**